data-raw/Global_IRI_Historical_ASAP_active.R

```diff
@@ -0,0 +1,324 @@
+#' **Title:** df_global_iriForecast_asapPhen.R
+#' Date: 2023-10-18
+#' **General Description:** 
+#' This script combines all historical IRI forecast data w/ all ASAP Phenology at the single global strata level 
+#' and outputs the results to tabular format.
+#' **General Purpose** Once the data is combined in tabular format it will be more efficient
+#' to experiment w/ thresholds and test performance
+#' **Main steps:**
+#'   Step 1. create unique id for each IRI pixel - convert to a polygon grid
+#'   Step 2. Aggregate %  active season data to each IRI poly grid
+#'   Step 3. Extract all historical IRI value (with grid ID)
+#'   Step 4. Merge extracted IRI and ASAP active season data by grid ID & season
+#'   Next Steps: once w decide on additional strata for analysis we can combine the polygon grid and the
+#'               new strata polygon.
+#' 
+#' 
+#' Note - to run in background:navigate to project root directory and run:
+#'   caffeinate -i -s Rscript data-raw/df_global_iri_active_thresholds.R
+
+
+
+# We initially ran this analysis using phenology data converted to binary using "End of Season" (EOS) as the definition
+# for ending the active season. We later decided it would be good to use "Senescence" (SEN) to create this rasters
+# Therefor, I can make this script flexible to take either raster input by just allowing the analyst to choose which set of rasters to use as input
+# all this does is change a single file path.
+
+phenology_data_definition <- c(
+  "EOS",
+  "SEN"
+)[2]
+
+
+# let user define what outputs to write, or rather just store in memory for testing
+write_grid <- c(T,F)[2] # already wrote this out
+
+write_merged_tabular_data <- c(T,F)[1]
+
+
+
+library(tidyverse)
```

Comment:
is your second comment here saying it is okay to leave? I figured it might make sense to have the user specified arguments before the library() so it is immediately flagged to the analyst and they dont think they can just run the script without consciously making a couple decisions?

I'm also happy to put it below library

Reply:
Yeah, I think that's my general thinking, the reasoning you outlined. But honestly would put up to you as I don't think it's critical at this point!